### PR TITLE
Core/Spells: Warlock Death Coil shouldn't be able to crit

### PR DESCRIPTION
**Pull request by r0m1ntik**
From TC [#26118](https://github.com/TrinityCore/TrinityCore/pull/26118)
[Proof](https://wow.gamepedia.com/Death_Coil_(warlock_ability))
```txt
It is a binary spell, and as such it can only hit or miss. Neither critical hits nor partial resists are possible.
```

## Changes Proposed:
-  Warlock [Death Coil](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=47860) shouldn't be able to crit

## Tests Performed:
-

## How to Test the Changes:

1. For a quick reproduction of the bug you can .additem 18970 ( [Ring of Critical Testing 2](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?item=18970)) a ring that gives you 50% crit .
2. Then cast Death Coil on a player / npc .
3. Check damage dealt / healing received .
## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/game/Spells/SpellMgr.cpp [hunk 1]
```diff
@@ -7082,10 +7082,15 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->SpellLevel = 0;
     });
 
-    // Drain Life - Bryntroll
+    // Drain Life - Bryntroll & Death Coil Warlock
```
Comment (Kitzunu) [inline src/server/game/Spells/SpellMgr.cpp:7085]: You can remove this comment as there are more spells here now with the description after the spells :)

Comment (r0m1ntik) [reply src/server/game/Spells/SpellMgr.cpp]: done 👍

**Review by Kitzunu — APPROVED** [review]

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@r0m1ntik nice, have you tested this in game?

**Review by Helias — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Review by Si1ker — APPROVED** [review]

**Pull request merged**